mautic/mautic
# [UI] Dropdown menus

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description
This PR is based [on another PR](https://github.com/mautic/mautic/pull/13735), where I introduce tokens to replace most of the UI variables, aiming to improve consistency in all components. **We need it to be merged before this one becomes available for code review.**

It changes dropdown styles and makes it compatible with dark theme.
Also fixes the too tiny size of clickable items on mobile.


Now when you hover over danger actions, have better visual feedback:

https://github.com/mautic/mautic/assets/116097999/79ec8ea1-6062-43fe-99d4-f67736a11c9c

Or navigating with your keyboard arrows

https://github.com/mautic/mautic/assets/116097999/439f7e9a-fef1-4b7b-be2a-e9d0e03ee341







| Before                                 | After
| -------------------------------------- | ---
|  ![image](https://github.com/mautic/mautic/assets/116097999/7e52cb4c-cf34-4c48-a971-1d97638364c5)                                      | ![image](https://github.com/mautic/mautic/assets/116097999/23dd8c1b-7939-4f16-9760-8b658b07e498)

![image](https://github.com/mautic/mautic/assets/116097999/72742f67-901d-4b0f-ac35-3632338b6e86)



---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Dashboard
3. Click on the caret to open the dropdown
